rts/Lua/LuaSyncedRead.cpp

```diff
@@ -8459,9 +8459,29 @@ int LuaSyncedRead::GetRadarErrorParams(lua_State* L)
 	} else {
 		lua_pushnumber(L, losHandler->GetBaseRadarErrorSize());
 	}
+	return 1;
+}
+
+
+// no docs, deprecated and same as GetAllyTeamRadarErrorParams
+int LuaSyncedRead::GetRadarErrorParams(lua_State* L)
+{
```

Comment:
Maybe print a deprecated message? (might need a helper for that), otherwise ppl likely won't notice and the real removing can take really long.